media-fonts/dotsies/dotsies-1.0~20181222.recipe

```diff
@@ -4,10 +4,10 @@ by hand, they take up a lot of unnecessary space. Your eyes have to move at \
 a frantic pace from left to right to read. The letters in Dotsies words \
 smoosh together, so words look like shapes! Get more screen space! Save \
 paper. Have a more comfortable reading experience."
-HOMEPAGE="http://dotsies.org/"
+HOMEPAGE="https://dotsies.org/"
 COPYRIGHT="2012 Craig Muth"
 LICENSE="CC-BY-4.0"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="http://dotsies.org/Dotsies.ttf#noarchive"
```

Comment:
What about SOURCE_URI?

Reply:
That one still slips away, sorry about that.